source/assets/js/components/fef-subnav.js

```diff
@@ -78,21 +76,9 @@ export class FefSubnav {
     }
 
     onResize() {
-        if (this.isWidthChangedSignificant()) {
-            this.init();
-            this.closeAllSubNavs();
-        }
-        this.width = window.width;
-    }
-
-    // If changed more than 15%
-    isWidthChangedSignificant() {
-        if (this.width < window.width) {
-            // More than 15% bigger than before
-            return ((widow.width * 100 / this.width) - 100) > 15;
-        } else {
-            // More than 15% smaller than before
-            return ((this.width * 100 / window.width) - 100) > 15;
+        let openNav = this.$element.find(`.${ITEM_OPEN_GROUP_CLASS}`);
+        if (openNav.length > 0) {
+            this.positionAndStretchSubNavGroup(openNav);
```

Comment:
hat das eine grosse Auswirkung auf die Performance? Die Logik ob es eine signifikante Änderung der Fensterbreite gab war schon sinnvoll anno dazumals, aber da war ein Bug drin. Ich glaube, man müsste `this.width` nur neu setzen wenn `this.isWidthChangedSignificant()` erfüllt war.

Jetzt wird's ja neu nicht mehr geschlossen beim Resize, ist das abgesprochen mit Design?

Reply:
Ja, hatte es mir bereits angeschaut ab ca. $(window).width() > als 1265px verschiebt es sich wieder. Hat keine Auswirkungen auf die Performance. Bin aber noch in Absprache mit dem Design.

Reply:
Design hat auch nichts dagegen.